docs/examples/viz_pbr_interactive.py

```diff
@@ -0,0 +1,228 @@
+from fury import actor, material, ui, window
+from fury.io import load_cubemap_texture
+from fury.data import fetch_viz_cubemaps, read_viz_cubemap
+from fury.utils import (normals_from_actor, tangents_to_actor,
+                        tangents_from_direction_of_anisotropy)
+
+
+def build_label(text, font_size=16, color=(1, 1, 1), bold=False, italic=False,
+                shadow=False):
+    label = ui.TextBlock2D()
+    label.message = text
+    label.font_size = font_size
+    label.font_family = 'Arial'
+    label.justification = 'left'
+    label.bold = bold
+    label.italic = italic
+    label.shadow = shadow
+    label.actor.GetTextProperty().SetBackgroundColor(0, 0, 0)
+    label.actor.GetTextProperty().SetBackgroundOpacity(0.0)
+    label.color = color
+    return label
```

Comment:
we use this in many places, maybe you should move it to `ui.utils`

Reply:
Yeah, that's what I thought too. I will do that.

Reply:
Many of the parameters were already in the UI component. I replaced all the calls to the function with calls to the UI component.